### ISSUE TITLE
Add additional metadata to Textual Inversion embedding training files

### ISSUE THREAD

**Issue by rockerBOO** [kohya-ss/sd-scripts#160]
Thanks for creating these scripts! Working with some tooling that reads in metadata included with the embedding files. Using this metadata to create charts and graphs for the embeddings. See https://github.com/Zyin055/Inspect-Embedding-Training and https://github.com/Zyin055/Inspect-Embedding-Training/pull/12

Specifically, the following metadata:

- name (token)
- sd-checkpoint
- step
- init word

Additionally:
- clip skip
- learning rate
- learning rate scheduler
- learning rate warm up steps
- batch size
- gradient accumulation steps
- color augments

Noting #112 can also be included. 

I plan on making these available and creating a PR. Any feedback, desires, objections or suggestions, please let me know.
